Lib/fontTools/subset/__init__.py

```diff
@@ -3297,33 +3363,33 @@ def _closure_glyphs(self, font):
                     self.glyphs.add(font.getGlyphName(i))
                 log.info("Added first four glyphs to subset")
 
-        if self.options.layout_closure and "GSUB" in font:
-            with timer("close glyph list over 'GSUB'"):
+        if "MATH" in font:
```

Comment:
the swapping of GSUB and MATH closure order should arguably go to a separate PR.. unless is really required for the changes at hand